examples/webvr_cubes.html

```diff
@@ -15,7 +15,7 @@
 			.menu {
 				position: fixed;
 				bottom: 20px;
-				right: 20px;
+				left: 20px;
```

Comment:
I initially had conditional styles that would put the menu on the left if iframed (`window.top !== window.self`), but I figured it made sense to just keep things in the same position regardless.

should probably use consistent menus for all WebVR examples, but that's not super serious.